app/views/layouts/application.html.erb

```diff
@@ -1,18 +1,28 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <title>Wizlist</title>
-    <%= csrf_meta_tags %>
-    <%= stylesheet_link_tag 'application', media: 'all' %>
-    <%= javascript_include_tag 'application' %>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+
+    <title>Wizlist — Your wishlists. Not your problem.</title>
+    <meta name="description" content="">
+    <%= csrf_meta_tags -%>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <%= stylesheet_link_tag 'application', media: 'all' -%>
   </head>
   <body>
+    <!--[if lt IE 8]>
+      <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
+
     <% if user_signed_in? %>
-      <%= link_to "Sign out", destroy_user_session_path, method: :delete %>
+      <%= render "shared/navigation" -%>
     <% end %>
 
-    <%= flash_messages %>
+    <%= yield -%>
 
-    <%= yield %>
+    <%= javascript_include_tag 'application' %>
+    <%= render "shared/flash_messages", flash: flash %>
```

Comment:
Not precisely. The whole `flash_messages` partial is only two JS scripts that are generated by erb. And they require jQuery already loaded. Putting the `<%= render "shared/flash_messages", flash: flash %>` anywhere above the `<%= javascript_include_tag 'application' %>` would cause an error, because the `$` which is a shortcut for `jQuery` would not yet defined.

The actual placement of the notification pop-up is irrelevant, because it's generated by the `bootstrap-notify` library.